### PR TITLE
Support Windows

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

File: template/data/config/environment.go.tmpl [hunk 1]
```diff
@@ -24,8 +24,9 @@ func init() {
 
 	if Environment == "production" {
 		gin.SetMode(gin.ReleaseMode)
-        os.MkdirAll("/tmp/{{.ProjectName}}/", 0700)
-		logFileName := fmt.Sprintf("/tmp/{{.ProjectName}}/%s-%d.log", Environment, time.Now().Unix())
+		tmpdir := filepath.Join(os.TempDir(), "{{.ProjectName}}")
+        os.MkdirAll(tmpdir, 0700)
```
Comment (mattn) [inline template/data/config/environment.go.tmpl]: oh

**Comment by shinriyo** [comment]
[LGTM](https://www.lgtm.in/i/MG7aQ2NXp)

[:+1:](https://www.lgtm.in/u/MG7aQ2NXp) [:-1:](https://www.lgtm.in/r/MG7aQ2NXp)
 
LGTM. Than you for pull request.

**Comment by shinriyo** [comment]
```
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
panic(0x153ce0, 0xc4200100e0)
	/usr/local/opt/go/libexec/src/runtime/panic.go:500 +0x1a1
github.com/dcu/gin-scaffold/command.(*InitCommand).Execute(0xc420014340, 0xc42000a140, 0x1, 0x1)
	/Users/shinriyo/src/github.com/dcu/gin-scaffold/command/init_command.go:57 +0x262
main.main()
	/Users/shinriyo/src/github.com/dcu/gin-scaffold/gin-scaffold.go:61 +0x14f
```

An error occurred

**Comment by shinriyo** [comment]
In my environment, I just minus one and resolved but..

```go:command/init_command.go
root = wd[len(p+"/src/")-1:]
```

**Comment by mattn** [comment]
Okay, will fix in soon.

**Comment by mattn** [comment]
Hmm, can't reproduce. could you please show me how to reproduce?

**Comment by mattn** [comment]
Ah, I figure out it. ignore this.

**Comment by shinriyo** [comment]
> could you please show me how to reproduce?

When, I did `init` command.

> Ah, I figure out it. ignore this.

Ok. I will check your pull-request. Thank you.

**Pull request merged**